web/src/app/shared/components/private-public-project/private-public-project.component.ts

```diff
@@ -15,45 +12,10 @@ import { IProject, ProjectModel } from '../../models/index.model';
   templateUrl: './private-public-project.component.html',
   styleUrls: ['./private-public-project.component.scss'],
 })
-export class PrivatePublicProjectComponent implements OnInit, OnDestroy {
+export class PrivatePublicProjectComponent {
 
-  private projectSubscription: Subscription;
-  public projects: ProjectModel[] = [];
-  public showTitle: boolean;
-  @Input() title: string = 'My Projects';
+  @Input() public projects: IProject[] = [];
```

Comment:
Do we need `public` here ?

Reply:
Oh no we dont, my mistake 🤦‍♂ . I will remove now.